### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by larispeight** [larispeight/cozy-calculator#3]
> look up if theres any other accessibility points i should be concerned about

- [x] vision
- [x] hearing

**Comment by larispeight** [comment]
"no aria is better than bad aria"
